pkg/block/block.go

```diff
@@ -32,6 +32,29 @@ const (
 	DRIVE_TYPE_VIRTUAL           // virtual drive i.e. loop devices
 )
 
+type PhysicalDiskType int
+
+const (
+	PHYSICAL_DISK_TYPE_UNSPECIFIED PhysicalDiskType = 0
+	PHYSICAL_DISK_TYPE_HDD PhysicalDiskType = 3
+	PHYSICAL_DISK_TYPE_SSD PhysicalDiskType = 4
+	PHYSICAL_DISK_TYPE_SCM PhysicalDiskType = 5
+)
+
+func (dt PhysicalDiskType) ToDriveType() DriveType {
+	switch dt {
+	case PHYSICAL_DISK_TYPE_UNSPECIFIED:
+		return DRIVE_TYPE_UNKNOWN
+	case PHYSICAL_DISK_TYPE_HDD:
+		return DRIVE_TYPE_HDD
+	case PHYSICAL_DISK_TYPE_SSD:
+		return DRIVE_TYPE_SSD
+	case PHYSICAL_DISK_TYPE_SCM:
+		return DRIVE_TYPE_UNKNOWN
+	}
+	return DRIVE_TYPE_UNKNOWN
+}
```

Comment:
can we move this into `pkg/block/block_windows.go` please, since it's Windows-specific? Also, let's name this `physicalDiskMediaType` (unexported) to denote that it's the WMI physical disk media type code.

Reply:
Ah yeah, I had it there before and then moved it for some reason... will move it back.